packages/docusaurus-theme-classic/src/routeAnnouncer.css

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#docusaurus-route-announcer-content {
```

Comment:
Using an ID selector here. Not sure if we necessarily want a class name here, or if we want to use inline styles.